lib/collection/response.js

```diff
@@ -63,7 +110,7 @@ _.assign(Response.prototype, /** @lends Response.prototype */ {
              * @type {String}
              * @deprecated use .reason()
              */
-            status: options.status ? options.status : httpReasons.lookup(options.code).name,
+            status: _.pluck(options.reason, options.status, httpReasons.lookup(options.code).name),
```

Comment:
`_.pluck` used to be a lodash 3 function which used to do something different. Sure we should use that name?